src/containers/internal/hash.d

```diff
@@ -51,10 +51,34 @@ size_t hashToIndex(const size_t hash, const size_t len) pure nothrow @nogc @safe
 	//
 	// It's amazing how much faster this makes the hash data structures
 	// when faced with low quality hash functions.
-	static if (size_t.sizeof == 8)
-		return (hash * 11_400_714_819_323_198_485UL) >>> (64 - bsr(len));
+	version (D_InlineAsm_X86_64)
+	{
+		asm @nogc
+		{
+			naked;
+			cmp RDI, 1;
```

Comment:
Take care to win64 , i think that register used are not the same (so RDI should be RCX maybe)

Reply:
Why did you have to remind me that Windows exists? I was hoping to forget.